db/warehouse/migrate/20230626005404_add2024_columns.rb

```diff
@@ -0,0 +1,39 @@
+class Add2024Columns < ActiveRecord::Migration[6.1]
+  def change
+    # Project
+    add_column :Project, :RRHSubType, :integer, null: true
+
+    # Client
+    # races
+    add_column :Client, :HispanicLatinaeo, :integer, null: true
+    add_column :Client, :MidEastNAfrican, :integer, null: true
+    add_column :Client, :AdditionalRaceEthnicity, :string, null: true
+    # genders
+    add_column :Client, :Woman, :integer, null: true # rename Female
+    add_column :Client, :Man, :integer, null: true # rename Male
+    add_column :Client, :NonBinary, :integer, null: true # rename NoSingleGender
+    add_column :Client, :CulturallySpecific, :integer, null: true
+    add_column :Client, :DifferentIdentity, :integer, null: true
+    add_column :Client, :DifferentIdentityText, :string, null: true
+
+    # Enrollment
+    add_column :Enrollment, :EnrollmentCoC, :string, null: true # limit? non-nullable!
```

Comment:
I think we'll need to allow nulls here, not every record has an associated EnrollmentCoC, and we could probably guess in 95% sod the situations, but there will be some where we can.t